src/dg/dg.cpp

```diff
@@ -2397,6 +2663,76 @@ real2 DGBase<dim,real,MeshType>::discontinuity_sensor(
     return eps;
 }
 
+//template<int dim, typename real, typename MeshType>
```

Comment:
What are those? Why are they all commentted? That function existed before?

Reply:
fixed by deleting. Was from an older version of the code